docs/docs/glossary.mdx

```diff
@@ -45,6 +45,10 @@ description: Glossary for all Rasa-related terms
 
   Incoming messages are processed by a sequence of components called a pipeline. A component can perform tasks ranging from entity extraction to intent classification to pre-processing.
 
+## [Conditional Response Variation](./responses.mdx#conditional-response-variations-experimental)
+
+   Response variation that can only be used when the current dialogue state satisfies some constraints as defined in the domain or responses files. If there's a match between the constraints and user filled slots, Rasa can use this variation.
```

Comment:
```suggestion
   Response variation that can only be used when the current dialogue state satisfies some constraints as defined in the domain or responses files. If there's a match between the constraints and the dialogue state, Rasa can use this variation.
```
Since the definition doesn't limit constraints to slots